src/platforms/default.rkt

```diff
@@ -16,26 +15,30 @@
       [(bool bool) not]
       [(bool bool bool) (and or)])))
 
-; machine floating-point operations (without conversions)
+;; machine floating-point operations
 (define machine-platform
   (with-terminal-cost ([binary64 64] [binary32 32])
-    (let ([rel-costs (cost-map
-                        [(PI E INFINITY NAN) 1]
-                        [(neg + - * /) 1]
-                        [(== != > < >= <=) 3])])
+    (let ([rel-costs (cost-map [(PI E INFINITY NAN) 1]
+                               [(neg + -) 2]
+                               [(*) 4]
+                               [(/) 10]
+                               [(== != > < >= <=) 4])])
       (platform-product
         [([real binary64] [bool bool]) (cost-map-scale 64 rel-costs)]
         [([real binary32] [bool bool]) (cost-map-scale 32 rel-costs)]
         (operator-set
           [(real) (PI E INFINITY NAN)]
-          [(real real) neg]
+          [(real real) (neg)]
           [(real real real) (+ - * /)]
           [(real real bool) (== != > < >= <=)])))))
 
-; libm operations
+
+;; libm operations
 (define libm-platform
   (with-terminal-cost ([binary64 64] [binary32 32])
-    (let ([rel-costs (cost-map #:default-cost 100)])
+    (let ([rel-costs (cost-map #:default-cost 100
+                               [(fabs) 2]
```

Comment:
I think this assumes `fabs` is lowered to the `and` trick, but that's fine.

Reply:
Yep pretty much.